tardis/gui/datahandler.py

```diff
@@ -63,7 +63,7 @@ def __init__(self, data, parent=None):
         parent : Node 
             Another node which is the parent of this node. The 
             root node has parent set to None.
-
+        
```

Comment:
Is this a necessary whitespace change? Doesn't matter much- just be careful what you commit.

Reply:
I did it just so that it would stay in the same indentation as the rest of the docstring, as it was one lower for some reason.